app/javascript/src/components/GameLogEntry.vue

```diff
@@ -165,7 +165,7 @@ export default {
         case 'playerGainsCash':
           return `${action.payload.player} gained ${action.payload.amount}m in taxes.`;
         case 'nationGainsTreasury':
-          return `${stringify(action.payload.nation.value)} gained ${action.payload.amount}m in taxes.`;
+          return `${stringify(action.payload.nation.value)} has changed by ${action.payload.amount}m in taxes.`;
```

Comment:
If nations cannot lose money, then this change shouldn't be made. Otherwise it should remain.